code/RealMeAuthenticator.php

```diff
@@ -25,51 +25,26 @@ protected static function on_register()
         $cache = SS_Cache::factory('RealMeAuthenticator');
 
         $cacheKey = 'RegisterCheck';
-        if (true === (bool) $cache->load($cacheKey)) {
+        if ((bool) $cache->load($cacheKey)) {
```

Comment:
@dhensby The cache content is a `boolean` represented as a `string`...

Line 51:
```
$cache->save('true', $cacheKey);
```

Testing for availability is not precisely congruent with the intended logic, I don't think. Do you have any thoughts on that?